microsoft/PowerToys
# [old settings] deprecate old settings

## Summary of the Pull Request

**What is this about:**
Remove the old settings

**What is include in the PR:** 
Remove the old settings from the main solution.
Remove the out binary from the signing process
Runner always runs the new settings

**How does someone test / validate:** 
Build the solution, build the installer and verify the new settings work as expected.
In the internal pipeline, verify the build completes successfully.

## Quality Checklist

- [x] **Linked issue:** #9417
- [ ] **Binaries:** Any new files are added/removed to WXS / YML
   - [x] Binary removed

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
